### ISSUE TITLE
"Unexpected token *" when used "**"

### ISSUE THREAD

**Issue by SomeoneOrGuesswho** [microsoft/vscode#179250]
!!! roblourens, GO AWAY, your answer does not solve the problem in any way !!!
!!! AND stop closing my report WITHOUT solving the issue !!!

Hi to all, except roblourens!
in VS Code used Code Runner v0.12

I try to run code in js file (js-file.js):
let x = 2;
console.log(x**3);

INSTEAD OF expected = 8, output:
js-file.js:2
console.log(x**3);
                      ^ (second *)

SyntaxError: Unexpected token *
at createScript (vm.js:56:10)
at Object.runInThisContext (vm.js:97:10)
at Module._compile (module.js:549:28)
at Object.Module._extensions..js (module.js:586:10)
at Module.load (module.js:494:32)
at tryModuleLoad (module.js:453:12)
at Function.Module._load (module.js:445:3)
at Module.runMain (module.js:611:10)
at run (bootstrap_node.js:387:7)
at startup (bootstrap_node.js:153:9)

[Done] exited with code=1 in 0.236 seconds

P.S.:
I) I have already reinstalled VS Code and Code Runner, but the issue is the same...
   WHAT and WHERE should I do to fix it ????????????????????????????????????????

II) The issue is NOT reproduced in browser AND on other computer!

III)  I repeat: I NEED STEPS TO FIX IT !

IV) not reproduced, if use such expration, for example, in Python (!!! the same VS Code where the issue with JS !!!)

**Comment by gjsjohnmurray** [comment]
/duplicate of #179147

Please do not open duplicate issues unless the previous one has been locked.

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! We figured it's covering the same as another one we already have. Thus, we closed this one as a duplicate. You can search for [similar existing issues](https://github.com/microsoft/vscode/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+). See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
